### PR TITLE
Peg openpyxl dependency to 2.4.9 to avoid non-backwards compatible api change

### PR DESCRIPTION
**Pull request by andrewgy8**
As mentioned [here](https://github.com/kennethreitz/tablib/issues/324) and [here](https://github.com/django-import-export/django-import-export/issues/762), openpyxl has changed there api for xlsx.  Currently when exporting to xlsx we receive a 500 error. 

Addresses #762

### CODE REVIEW

File: import_export/formats/base_formats.py [hunk 1]
```diff
@@ -230,6 +230,9 @@ def create_dataset(self, in_stream):
             dataset.append(row_values)
         return dataset
 
+    def export_data(self, dataset, **kwargs):
+        return self.get_format().export_set(dataset, **kwargs)
+
```
Comment (bmihelac) [inline import_export/formats/base_formats.py]: Is this necessary? It looks like it does not change what would parent method do.

**Comment by andrewgy8** [comment]
I would say that would be the _best_ course of action. But it does not seem like we can be sure when tablib will have this fixed/released.  As you can see in[ this issue](https://github.com/kennethreitz/tablib/issues/329), there seems to be some worry from the community about the maintenance and governance in general.

I would say the _next best_ course of action, is for ourselves to be proactive (this PR) AND watch closely the progress of the issue in tablib that addresses this error.  And it might be a healthy exercise to watch the state of tablib in general.

**Comment by lubo** [comment]
@andrewgy8 is right, we should stick to versions that are known to work. Although I'd stick to openpyxl 2.4 series, rather than to 2.4.9 in particular. Something like `openpyxl>=2.4,<2.5`.

**Comment by andrewgy8** [comment]
@bmihelac I have updated this PR. What do you think? Shall we peg and wait?

**Comment by dv321** [comment]
This is causing exports to break for me unless I peg the openpyxl version, and since the resolution of the tablib issue seems unclear I'd humbly recommend that this be merged so the library doesn't come broken out of the box, at least until the tablib fix goes through.

**Comment by aparakian** [comment]
Any update on this? That would be great to have this merged since it seems to solve the issue quite well.

**Comment by andrewgy8** [comment]
@aparakian Still in a holding pattern for a member to merge. 

@bmihelac Any chance we can merge this soon?

**Comment by bmihelac** [comment]
Merged, @andrewgy8 thanks for patch!

**Comment by aparakian** [comment]
Thank you @bmihelac for the reactivity! ⭐️

**Comment by andrewgy8** [comment]
No worries @bmihelac 👍 

I'd like to help out as much as I can as we use this package quite extensively. Ill keep an eye out for other patches I can submit :wink:

**Pull request merged**